### DISCUSSION TITLE
Payment for Organization

### DISCUSSION THREAD

**Discussion by danielmeyer15** [getredash/redash#6056] [Q&A] [answered]
how to used redash for organization (payable)?
if i see on https://redash.io/terms in point number 10, redash can used for organization (payable)

**Comment by justinclift** [comment, marked as answer]
That's probably just a left over piece of text, from when Redash used to have an online hosted service (SaaS).  That was before they were [acquired by Databricks](https://www.databricks.com/blog/2020/06/24/welcoming-redash-to-databricks.html), and shut the online service down.

**Reply by danielmeyer15** [reply]
thank you for your answer @justinclift , but we have a issued in redash in security, is databricks supported?

**Reply by justinclift** [reply]
@arikfr ?
